source/2020-05-22-the-ember-times-issue-149.md

```diff
@@ -0,0 +1,146 @@
+---
+title: The Ember Times - Issue No. XXS
+author: Matthew Roderick, the crowd
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
+alias : "blog/2020/xx/xx-the-ember-times-issue-XXX.html"
+responsive: true
+---
+
+<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
+
+<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+
```

Comment:
```suggestion
Cal Woolgar's video series 🐺
```

Was wondering why you went with that emoji? Not a problem but could use 🎥 more relevant?